src/packages/sdk/src/cli/hashes.ts

```diff
@@ -0,0 +1,33 @@
+import { getSchemaPath } from './getSchema'
+import { arg } from './utils'
+import crypto from 'crypto'
+
+/**
+ * Get a unique identifier for the project by hashing
+ * the directory with `schema.prisma`
+ */
+export async function getProjectHash(): Promise<string> {
+  const args = arg(process.argv.slice(3), { '--schema': String })
+
+  let projectPath = await getSchemaPath(args['--schema'])
```

Comment:
Because there is a throw here https://github.com/prisma/prisma/blob/902d4384fbadc9e27635c76e43f1bd4d25c0ba7e/src/packages/sdk/src/cli/getSchema.ts#L24-L27

That means `getProjectHash` won't work if `args['--schema']` is invalid.

In the end from the checkpoint perspective ignoring `args['--schema']` could be easier / better? @2color

Reply:
I guess the error would actually be thrown before reaching this anyway :D

Reply:
As discussed directly: 
If the --schema is invalid, I thought it won’t reach the checkpoint.check() call.